src/python/pants/backend/terraform/lint/tfsec/rules.py

```diff
@@ -30,30 +37,36 @@ async def run_tfsec(request: TfSecRequest.Batch, tfsec: TFSec, platform: Platfor
                 downloaded_tfsec.digest,
                 sources.snapshot.digest,
                 config_file.snapshot.digest,
+                report_directory,
             )
         ),
     )
 
     computed_args = []
     if tfsec.config:
-        computed_args = [f"--config-file={tfsec.config}"]
+        computed_args.append(f"--config-file={tfsec.config}")
+
+    if tfsec.report_name:
+        computed_args.append(f"--out=reports/{tfsec.report_name}")
```

Comment:
Should this use `REPORT_DIR` to match what we pass to "output_directories"?
```suggestion
        computed_args.append(f"--out={REPORT_DIR}/{tfsec.report_name}")
```

Reply:
Ah oops, good catch. Forgot to template that out when I changed the approach.